stable_baselines/a2c/a2c.py

```diff
@@ -143,8 +179,22 @@ def _train_step(self, obs, states, rewards, masks, actions, values):
             td_map[self.train_model.states_ph] = states
             td_map[self.train_model.masks_ph] = masks
 
-        policy_loss, value_loss, policy_entropy, _ = self.sess.run(
-            [self.pg_loss, self.vf_loss, self.entropy, self.apply_backprop], td_map)
+        if update % 10 == 9:
```

Comment:
Why 10? You should define it as a variable
```python
(update + 1) % 10 = 0
```

+ missing comments of what you are doing